cluster-autoscaler/FAQ.md

```diff
@@ -933,7 +934,9 @@ Events:
 ```
 ### My cluster is below minimum / above maximum number of nodes, but CA did not fix that! Why?
 
-Cluster Autoscaler will not scale the cluster beyond these limits, but does not enforce them. If your cluster is below the minimum number of nodes configured for Cluster Autoscaler, it will be scaled up *only* in presence of unschedulable pods.
+Cluster Autoscaler will not scale the cluster beyond these limits, but does not enforce them. By default, if your cluster is below the minimum number of nodes configured for Cluster Autoscaler, it will be scaled up *only* in presence of unschedulable pods.
```

Comment:
nit: "but does not enforce them" isn't true anymore for min size with this change. Maybe rephrase this somehow?

Reply:
Right, I will rephrase the whole paragraph.